src/features/integration/components/form/CaseInformation.tsx

```diff
@@ -13,7 +13,9 @@ const CaseInformation: React.FunctionComponent<any> = (props) => {
     const caseInformationFields: IInputField[] = [
         {input: INPUT_TYPE.TEXT_FIELD, label: "Navn", formValue: "name", required: props.validation, error:errors.name},
         {input: INPUT_TYPE.TEXT_FIELD, label: "Beskrivelse", formValue: "description", required: props.validation, error:errors.description},
+        {input: INPUT_TYPE.TEXT_FIELD, label: "Skjemaleverandør", formValue: "sourceApplication", required: props.validation, error:errors.sourceApplication},
         {input: INPUT_TYPE.DROPDOWN, label: "Skjema", value: props.watch("sourceApplicationIntegrationId"), formValue: "sourceApplicationIntegrationId", dropDownItems: forms},
+        {input: INPUT_TYPE.TEXT_FIELD, label: "orgId", formValue: "orgId", required: props.validation, error:errors.orgId},
```

Comment:
orgId = OrganizationId  ? Is this visible to the end user?

Reply:
no, this is not reflected in any of the form field. internal model variable. 
organisationId for the applicant is set in the applicantData.organisationNumber field